### ISSUE TITLE
Tracking pixel support /img interception

### ISSUE THREAD

**Issue by kolodiy** [chmanie/wdio-intercept-service#79]
Hi,
As I understand currently there is no possibility to intercept img requests. 
@chmanie are there any plans to add this functionality? This would be really useful for verification of analytics!

**Comment by chmanie** [comment]
Yeah this is probably related to #47. I guess we could do it but I don’t have much time at the moment to implement something like this

**Comment by kolodiy** [comment]
If anyone bumps into this - I found a workaround by using: https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntries and filtering returned entries.
